common-types/index.d.ts

```diff
@@ -86,3 +86,28 @@ interface EventProofImage {
   readonly url: string;
   readonly fileName: string;
 }
+
+interface CandidateDeciderRating {
+  readonly reviewer: IdolMember;
+  readonly rating: number;
+}
+
+interface CandidateDeciderComment {
+  readonly reviewer: IdolMember;
+  readonly comment: string;
+}
+
+interface CandidateDeciderCandidate {
+  readonly responses: any[];
```

Comment:
I see a GitHub warning for this type. Why do we choose to use `any[]`?

Reply:
This field is the responses from the candidates so the questions depend from form to form. I wasn't sure if there was a better way to do this to avoid `any`

Reply:
got it! In this case, `any` seems suitable

Reply:
These *should* be all strings in my experience, but I recommend creating a union type rather than using `any` to narrow down all possibilities if we add something other than string.